firefox-ios/Shared/Strings.swift

```diff
@@ -172,6 +172,11 @@ extension String {
                 tableName: "Bookmarks",
                 value: "Edit Bookmark",
                 comment: "Label on the top of the `Edit Bookmarks` screen.")
+            public static let EditBookmarkDesktopBookmarksLabel = MZLocalizedString(
+                key: "Bookmarks.Menu.EditBookmarkDesktopBookmarksLabel.v135",
```

Comment:
It would be for 136 no? We are releasing for 136 and I think 135 string freeze was already done?
```suggestion
                key: "Bookmarks.Menu.EditBookmarkDesktopBookmarksLabel.v136",
```

Reply:
Yes, string freeze for 135 has passed, but soft code freeze/v135 branch creation hadn't happened (until earlier today) so I originally marked this as v135 since that's the release it was going in, not necessarily when it would be translated. Thought this would be fine to add after string freeze since we don't need translations until v136.

Do you know if translators use this version number for anything? If not, then correct me if I am missing something, but I think it would make most sense that if you are (in this rare scenario) adding strings between string freeze and soft code freeze (for a future version eg v136) that we would still use the current release version (eg v135)

Now that `release/v135` has been created, I will update the versions in all of my open PR's to v136!

Reply:
Translators are not using this version number, but I think it's nice to give us an idea of when it was translated. It could help us if we need to remove/clean up some strings to know when it was exported for translations 🤷‍♀️  It's more of a nit than something serious though